incubator/hnc/internal/reconcilers/hierarchy_config.go

```diff
@@ -170,6 +174,14 @@ func (r *HierarchyConfigReconciler) onMissingNamespace(log logr.Logger, nm strin
 	}
 }
 
+// removeExcludedNamespaceLabel removes the `hnc.x-k8s.io/excluded-namespace`
+// label from the namespace.
+func (r *HierarchyConfigReconciler) removeExcludedNamespaceLabel(nsInst *corev1.Namespace) {
+	lbs := nsInst.Labels
```

Comment:
It's not a bad idea to log something here, since this is definitely unusual.

```
if _, found := lbs[api.LabelExcludedNamespace]; found {
  log.Info("Illegal excluded-namespace label found; removing")
}
```